go/vt/sqlparser/sql.y

```diff
@@ -4238,6 +4268,79 @@ reserved_table_id:
     $$ = NewTableIdent(string($1))
   }
 
+infile_opt:
+  { $$ = string("") }
+| INFILE STRING
+  { $$ = string($2)}
+
+local_opt:
+  { $$ = BoolVal(false) }
+| LOCAL
+  { $$ = BoolVal(true) }
+
+enclosed_by_opt:
+  {
+    $$ = ""
+  }
+| optionally_opt ENCLOSED BY STRING
+  {
+    $$ = "'" + string($4) + "'"
+  }
+
+optionally_opt:
+  {
+    $$ = ""
+  }
+| OPTIONALLY
+  {
+    $$ = " optionally"
```

Comment:
This token should be a boolean value. The "optoinally" option is either set or not